### DISCUSSION TITLE
add forwarded messages directly to the bot from the admin to spam

### DISCUSSION THREAD

**Discussion by lk29** [umputun/tg-spam#108] [Q&A] [answered]
add a function so that messages sent from the admin(s) directly to the bot receive spam status and are added to the spam message database

**Comment by umputun** [comment]
need more details. Currently messages forwarded by admin to spam group acting exactly like this, not sure why you may need this functionality to be duplicated

**Comment by umputun** [comment]
@lk29 any updates?

**Comment by lk29** [comment]
> need more details. Currently messages forwarded by admin to spam group acting exactly like this, not sure why you may need this functionality to be duplicated
> 
> 

Spam group? 
I'm probably not using the bot correctly.

Can you tell me where I can read about setting up "spam group"

**Comment by umputun** [comment, marked as answer]
https://github.com/umputun/tg-spam?tab=readme-ov-file#admin-chatgroup
